operator/bundle_helpers/patch-csv.py

```diff
@@ -91,6 +96,8 @@ def parse_args():
     parser.add_argument("--replace-rbac-proxy", required=False, metavar='original-tag:replacement-image',
                         nargs='+', help='Replacement directives for the RBAC proxy image',
                         default=[])
+    parser.add_argument("--downstream-arches", action='store_true',
+                        help='Add labels for downstream architectures')
```

Comment:
```suggestion
    parser.add_argument("--extra-arch", action='append', required=False,
                        help='Mark the operator as supporting an additional architecture')
```

Reply:
Testing showed that this also needs `default=[]` for the case when the flag is not passed.